<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addTtlvByte</class>
 <widget class="QWidget" name="addTtlvByte">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>字节长度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MySpinBox" name="spinBox_len">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>240</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QCheckBox" name="checkBox_event">
     <property name="text">
      <string>配置事件</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_name">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>事件名称</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="event_name">
     <property name="maxLength">
      <number>64</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_type">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>事件类型</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QComboBox" name="event_type">
     <item>
      <property name="text">
       <string>信息</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>告警</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>故障</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_condition">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>触发条件</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>字节流</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>字符串</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="register_value"/>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QWidget" name="widget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MySpinBox</class>
   <extends>QSpinBox</extends>
   <header location="global">myspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
